packages/pcd/gpc-pcd/src/GPCPCDPackage.ts

```diff
@@ -290,19 +303,121 @@ function validateInputPOD(
     if (podConfig === undefined) {
       params.notFoundMessage = `The proof configuration does not contain this POD.`;
       return false;
-    } else {
-      const entries = Object.keys(podConfig.entries);
-      // Enumerate POD entries
-      const podEntries = podPCD.pod.content.asEntries();
-      // Return true iff all elements of `entries` are keys of `podEntries`
-      return entries.every((entryName) => podEntries[entryName] !== undefined);
+    }
+    const configEntries = Object.keys(podConfig.entries);
+
+    // Enumerate POD entries
+    const podEntries = podPCD.pod.content.asEntries();
+    // Return false if some entry in the config is not in the POD
+    if (
+      configEntries.some((entryName) => podEntries[entryName] === undefined)
+    ) {
+      return false;
+    }
+
+    // Alist of (entry name, membership list name) pairs.
```

Comment:
Is "Alist" a thing?  I thought it was a typo of "A list" but then I saw it appear in multiple variable names.  Google search is not very helpful but digging through the possible results maybe you're referring to an Association List (https://www.gnu.org/software/emacs/manual/html_node/elisp/Association-List-Type.html)?

This isn't a term I've heard before, so I suspect other readers may not be familiar with it either.  Maybe this is just `listMemberships`?

Alternatively, if you're looking for associativity, why is this a list of pairs rather than a Map or object?